modules/ec2-instance/templates/userdata.sh.tmpl

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo "Hello user data"
```

Comment:
Shouldn't this include some variable and better support templating?

e.g. 
```bash
#!/bin/bash

echo "Begin userdata.sh"
${var.userdata}
```

Reply:
a lot of this was taken from the `zscaler` component, so I had stripped what wasnt necessary for a basic ec2 instance. But that's a good idea. Will add that in